app/controllers/change_requests_controller.rb

```diff
@@ -18,7 +18,17 @@ def create_change_request
     elsif params[:service_id]
       @change_request = ServiceChangeRequest.create(object_id: params[:service_id], resource_id: Service.find(params[:service_id]).resource_id)
     elsif params[:address_id]
-      @change_request = AddressChangeRequest.create(object_id: params[:address_id], resource_id: Address.find(params[:address_id]).resource_id)
+      if params[:change_request][:action]
+        action_code = params[:change_request][:action]
+        if ['remove'].include?(action_code)
```

Comment:
Is this just the same as `if action_code == 'remove'`? My assumption would be that maybe you were originally trying to match against multiple different action_code strings but then later deleted the other strings, leaving just a single-element array here.

Could you write this in the more usual `action_code == 'remove'` style?

Reply:
@richardxia yes and yes!